server/src/main/java/org/candlepin/resource/ProductResource.java

```diff
@@ -286,10 +300,11 @@ public void deleteProduct(@PathParam("product_id") String productId) {
     @GET
     @Path("/{product_id}/statistics")
     @Produces(MediaType.APPLICATION_JSON)
-    public List<Statistic> getProductStats(@PathParam("product_id") String productId,
-                            @QueryParam("from") String from,
-                            @QueryParam("to") String to,
-                            @QueryParam("days") String days) {
+    public List<Statistic> getProductStats(
+        @Verify(Product.class) @PathParam("product_id") String productId,
```

Comment:
A method without a @Verify param and without @SecurityHole becomes super-admin only, and I think we want to preserve that particularly in these global resource classes, introducing @Product verify might actually be opening this up to callers who shouldn't be able to add content to a product or get product stats, I think what we want here is to remove @Verify and take this right back to super admin only. (which is what Katello would operate as when it makes these calls)